src/plugins/chatwizard/chatwizard.js

```diff
@@ -170,6 +174,9 @@ var componentName = "wb-chtwzrd",
 		// Initiate chat wizard bubble
```

Comment:
At line 165 wher you have: `$minimize = $( ".minimize", $container ),` don't forget to also add `$reset = $( ".reset", $container ),`, and then to change `$firstTabStop = $minimize,` for `$firstTabStop = $reset,`